tests/linearizability/model.go

```diff
@@ -47,8 +47,8 @@ type EtcdResponse struct {
 }
 
 type EtcdState struct {
-	Key            string
-	PossibleValues []ValueRevision
+	Revision  int64
+	KeyValues map[string]string
 }
 
 type ValueRevision struct {
```

Comment:
Why do you still keep `ValueRevision`?

Reply:
Based on https://github.com/etcd-io/etcd/pull/14924#discussion_r1045433830, we still need to keep `ValueRevision`

Reply:
Removed.